include/mbedtls/des.h

```diff
@@ -44,7 +44,8 @@
 #define MBEDTLS_ERR_DES_INVALID_INPUT_LENGTH              -0x0032  /**< The data input has an invalid length. */
 
 /* MBEDTLS_ERR_DES_HW_ACCEL_FAILED is deprecated and should not be used. */
-#define MBEDTLS_ERR_DES_HW_ACCEL_FAILED                   -0x0033  /**< DES hardware accelerator failed. */
+#define MBEDTLS_ERR_DES_HW_ACCEL_FAILED                   -0x0033  /**< DES hardware accelerator failed.      */
```

Comment:
Minor: please don't try to align end-of-comment markers here (this is an exception to our usual style just for error codes, similar to the fact that we don't care about line length just here).

Reply:
Yes.